### PR TITLE
initial migration of zabbix module namespace from ansible-base

### PR DESCRIPTION
**Pull request by D3DeFi**
##### SUMMARY
1st step of migration of Zabbix module namespace into a dedicated collection

##### ISSUE TYPE
New Module Pull Request

##### COMPONENT NAME
\*zabbix\*

##### ADDITIONAL INFORMATION
[No sure if I understood](https://miro.medium.com/max/455/1*snTXFElFuQLSFDnvZKJ6IA.png) instructions perfectly. I've cloned forks of both `community.{general,zabbix}` and manually `cp`-ied files to this collection. Old paths containing `community.general` were changed to new. Should I somehow transfer commit history as well?

I am thinking about adopting "orphaned" [contrib/inventory/zabbix.py](https://github.com/ansible/ansible/blob/devel/contrib/inventory/zabbix.py) as well. Can this contribution script be placed into `plugins/inventory` as it is an external invnetory script or does it need a different placement? 

Tests will not work for now, but that is a subject to change.

### CODE REVIEW

**Comment by D3DeFi** [comment]
@gundalow can you please acknowledge that this is the way I was supposed to transfer the files?

Next step would be to open PR with `git rm`-ied files from `modules/monitoring/zabbix` and any reference to them in `community.general`?

File: tests/sanity/ignore-2.10.txt [hunk 1]
```diff
@@ -0,0 +1,30 @@
+scripts/inventory/zabbix.py future-import-boilerplate
+scripts/inventory/zabbix.py metaclass-boilerplate
+plugins/modules/monitoring/zabbix/zabbix_action.py validate-modules:doc-choices-do-not-match-spec
```
Comment (gundalow) [inline tests/sanity/ignore-2.10.txt]: In tests/sanity/ignore-2.10.txt and tests/sanity/ignore-2.9.txt could you please do:
```suggestion
plugins/modules/zabbix_action.py validate-modules:doc-choices-do-not-match-spec
```

**Comment by D3DeFi** [comment]
Should be good know, but I doesn't want to merge it without your approval @gundalow :)
Opening PR against `community.general` in a sec

**Comment by gundalow** [comment]
Looks good, thanks.

**Comment by D3DeFi** [comment]
> 
> 
> Looks good, thanks.

Thanks for your help!

**Pull request merged**